Make ng-boosted compatible with last version of angular cli (12.2.3)

Hello,

I installed the last version of angular/cli (12.2.3),

then I created a new project, inside I tried to install ng-boosted, but I got an error related to a dependency version mismatch for angular-cli, ng-boosted wants angular/cli 12.1.1, but a more recent version of angular/cli is 12.2.3 : 

```
$ npm install --save ng-boosted

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: test2@0.0.0
npm ERR! Found: @angular/common@12.2.3
npm ERR! node_modules/@angular/common
npm ERR!   @angular/common@"~12.2.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/common@"~12.1.1" from ng-boosted@10.0.0
npm ERR! node_modules/ng-boosted
npm ERR!   ng-boosted@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

Please update the ng-boosted package, in order to make it compatible with the last version of angular/cli.

Thanks

--- a/projects/ng-boosted/package.json
+++ b/projects/ng-boosted/package.json
@@ -1,21 +1,21 @@
 {
   "name": "ng-boosted",
   "version": "10.0.0",
-  "description": "Orange Boosted with Bootstrap 4 and Angular",
+  "description": "Orange Boosted with Bootstrap 5 and Angular",
   "keywords": [
     "boosted",
     "angular",
     "typescript"
   ],
   "author": "Orange",
-  "homepage": "https://orange-opensource.github.io/Orange-Boosted-Angular",
+  "homepage": "https://ng-boosted.orange.com",
   "license": "MIT",
   "peerDependencies": {
-    "@angular/common": "~12.1.1",
-    "@angular/core": "~12.1.1",
-    "@angular/forms": "~12.1.1",
-    "@angular/localize": "~12.1.1",
-    "rxjs": "~6.6.7",
+    "@angular/common": "^12.1.1",
+    "@angular/core": "^12.1.1",
+    "@angular/forms": "^12.1.1",
+    "@angular/localize": "^12.1.1",
+    "rxjs": "^6.6.7",
     "node-sass": "^4.14.1"
   },
   "dependencies": {
@@ -31,7 +31,7 @@
     "url": "https://github.com/Orange-OpenSource/Orange-Boosted-Angular/issues"
   },
   "engines": {
-    "node": ">= 4.2.1",
-    "npm": ">= 3"
+    "node": ">= 14.17.5",
+    "npm": ">= 6.14.11"
   }
 }

